src/conn.c

```diff
@@ -311,7 +341,10 @@ int conn__start(struct conn *c,
 	c->transport.data = c;
 	c->uv_transport = uv_transport;
 	c->close_cb = close_cb;
-	gateway__init(&c->gateway, config, registry, raft, seed);
+	uv_timer_init(loop, &c->defer);
+	c->defer_cb = NULL;
+	c->defer_arg = NULL;
```

Comment:
Just to triple-check, there is always going to be a maximum of one exec request per connection, right? So this fields are never going to be overwritten while they are in use.

Reply:
Yep.